srcpkgs/xpadneo/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'xpadneo'
+pkgname=xpadneo
+version=0.9.1
+revision=1
+depends="dkms bluez"
+short_desc="Bluetooth driver for Xbox One Wireless Controller"
+maintainer="Joshua Krämer <joshua@kraemer.link>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/atar-axis/xpadneo"
+distfiles="https://github.com/atar-axis/xpadneo/archive/v${version}.tar.gz"
+checksum=1f3617b0689d93877a706a52037649d6cad011cf70491b5b43da015a02312058
+dkms_modules="hid-xpadneo ${version}"
+
+do_install() {
+	vmkdir usr/src/hid-xpadneo-${version}
+	vcopy "hid-xpadneo/*" usr/src/hid-xpadneo-${version}
+	mv ${DESTDIR}/usr/src/hid-xpadneo-${version}/dkms.conf.in ${DESTDIR}/usr/src/hid-xpadneo-${version}/dkms.conf
+	sed -i -e "s/@DO_NOT_CHANGE@/${version}/" ${DESTDIR}/usr/src/hid-xpadneo-${version}/dkms.conf
```

Comment:
Please use `vsed` and install the file with `vinstall`.

Reply:
I have updated the template as discussed on IRC.